src/parser/jobs/gnb/modules/OGCDDowntime.tsx

```diff
@@ -0,0 +1,16 @@
+import ACTIONS from 'data/ACTIONS'
+import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
+
+export default class AbilityDowntime extends CooldownDowntime {
+	allowedDowntime = 2500
+	trackedCds = [
+		ACTIONS.DANGER_ZONE.id,
+		ACTIONS.BLASTING_ZONE.id,
+		ACTIONS.ROUGH_DIVIDE.id,
+		ACTIONS.BOW_SHOCK.id,
+	]
+	// account for charge system? big question mark if this even works
```

Comment:
probably need to sync up with @rexatlanti about that; most jobs have charges somewhere so it'll need to be tracked in some manner.